setup.py

```diff
@@ -90,7 +90,8 @@ def run_setup():
                 'parsely_redshift = parsely_raw_data.redshift:main',
                 'parsely_s3 = parsely_raw_data.s3:main',
                 'parsely_stream = parsely_raw_data.stream:main',
-                'parsely_schema = parsely_raw_data.docgen:main'
+                'parsely_schema = parsely_raw_data.docgen:main',
+                'parsely_dbt_redshift = parsely_raw_data/dbt/parsely_dpl_redshift.redshift:main'
```

Comment:
We were chatting about this already earlier, but so it doesn't get forgotten -- these slashes should be dots.